docker-compose.yml

```diff
@@ -9,28 +9,15 @@ services:
       POSTGRES_USER: spoke
     volumes:
       - postgres:/var/lib/postgresql/data
+    ports:
+      - 5432:5432
   redis:
     image: redis:alpine
     restart: always
     volumes:
       - redis:/data
-  spoke:
```

Comment:
Will this work with the heroku deploy? Maybe Heroku just uses the `Dockerfile`?

Also, I am super-naive about docker, but doesn't this delete the spoke app itself here?  How does spoke get run within Docker?  Or the idea is that Dockerfile is run 'automatically' and the services here are supplemental?

I would also ++ @ibrand 's request for maybe a "HOWTO-run-docker.md" file -- then we could link to it -- maybe even as the default instructions?

Reply:
@schuyler1d this will not mess with the Heroku deploy cause [it does use the Dockerfile](https://github.com/MoveOnOrg/Spoke/blob/main/heroku.yml#L3)

I'm also not so Docker knowledgeable, so not sure about your question there

Reply:
Yeah, it doesn't impact heroku at all. Heroku uses the Dockerfile to build an image for Spoke, whereas the docker-compose file is just used to run the standard postgres and redis images for local development.